hathor/indexes/memory_info_index.py

```diff
@@ -28,8 +31,11 @@ def __init__(self):
         self._first_timestamp = 0
         self._latest_timestamp = 0
 
-    def init_start(self, indexes_manager: 'IndexesManager') -> None:
+    @inlineCallbacks
+    def init_start(self, reactor: 'Reactor', indexes_manager: 'IndexesManager') -> Generator[Any, Any, None]:
```

Comment:
Same here, it doesn't need to use `@inlineCallbacks`.

Reply:
Done! Thanks!